### PR TITLE
Create license file.

### PR DESCRIPTION
**Pull request by frslm**
Since the setup file lists the license as "MIT", I propose adding this GitHub-generated MIT license file.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Michael Woods
```
Comment (sandrotosi) [inline LICENSE:3]: i dont think this is accurate: Michael made the first commit in 2014, so it seems natural he'd hold the copyright from that year up until 2019 (but i guess we can update it to 2020 no?)

I'll let @vgm64 comment for clarification

Comment (vgm64) [reply LICENSE:3]: Copyright has never expressly been transferred by any author of the repo, so copyright shouldn't fall to me. Suggestion:

`Copyright (c) 2014-present, gmplot contributors`

Comment (sandrotosi) [reply LICENSE:3]: The only way i manage to remember what copyright and license are is: license is how you can use the software, copyright is who decides the license.

so i think a more appropriate notice would be

```
Copyright (c) 2014-2019 Michael Woods
Copyright (c) 2020 gmplot contributors
```
(you cannot use `present` in copyright)

Comment (frslm) [reply LICENSE:3]: @sandrotosi's suggestion makes sense to me. @vgm64 if it's fine with you too, I'll put up a commit to change it to that.

Comment (vgm64) [reply LICENSE:3]: Works for me.

Comment (frslm) [reply LICENSE]: Done in 48c655b7fd1a4f529181eb1ef3df210415fcc97c.

**Pull request merged**